src/shogun2-core/src/main/java/de/terrestris/shogun2/service/ModelDescriptionService.java

```diff
@@ -0,0 +1,35 @@
+package de.terrestris.shogun2.service;
+
+import java.io.IOException;
+
+import org.springframework.stereotype.Service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.module.jsonSchema.JsonSchema;
+import com.fasterxml.jackson.module.jsonSchema.JsonSchemaGenerator;
+
+import de.terrestris.shogun2.model.PersistentObject;
+
+/**
+ * Service class to describe classes model.
+ *
+ * terrestris GmbH & Co. KG
+ * @author Kai Volland
+ * @date 04.09.2017
+ *
+ */
+@Service("modelDescriptionService")
+public class ModelDescriptionService<E extends PersistentObject> {
+	
+	public JsonSchema getJsonSchema(Class<?> clazz) throws IOException {
+		ObjectMapper mapper = new ObjectMapper();
```

Comment:
There is a project-specific `ObjectMapper` that is available as bean (if archetype has been used) and should be used here:

https://github.com/terrestris/shogun2/blob/master/src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/__artifactId__-servlet.xml#L54